src/load.cpp

```diff
@@ -289,28 +299,32 @@ void LoadOutput::state_machine()
             }
             break;
         case LOAD_STATE_OFF_LOW_SOC:
-            // wait at least one hour
+            // wait configured time
             if (time(NULL) - lvd_timestamp > lvd_recovery_delay) {
                 state = LOAD_STATE_DISABLED; // switch to normal mode again
             }
             break;
         case LOAD_STATE_OFF_OVERCURRENT:
-            // wait some time
+            // wait configured time
             if (time(NULL) - overcurrent_timestamp > overcurrent_recovery_delay) {
+                log_data.error_flags &= ~(1 << ERR_LOAD_OVERCURRENT);
+                log_data.error_flags &= ~(1 << ERR_LOAD_VOLTAGE_DIP);
                 state = LOAD_STATE_DISABLED;   // switch to normal mode again
             }
             break;
         case LOAD_STATE_OFF_OVERVOLTAGE:
             if (port->voltage < (port->sink_voltage_max - 0.5) &&
```

Comment:
Does it make sense if we wait for 1s before clearing the flag as we did for setting the same? Or do we have a reason for not doing this?

Reply:
I think we don't need the delay here, as short overvoltage peaks are quite likely (e.g. during disconnection of an inductive load). Undervoltage peaks are not really possible if the load is disconnected already.

Reply:
Agreed